Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -128,7 +119,7 @@ theorem lfpApprox_mono_left : Monotone (lfpApprox : (α →o α) → _) := by
       simp only [OrderHom.toFun_eq_coe]
       exact g.monotone (ih i' h_lt)
 
-theorem lfpApprox_mono_mid : Monotone (lfpApprox f) := by
+theorem lfpApprox_mono_mid (f : α →o α) : Monotone (lfpApprox f) := by
   intro x₁ x₂ h a
   induction a using Ordinal.induction with
   | h i ih =>
```

Comment:
Maybe you could also get rid of the `constructor` here and use `use le_sup_of_le_left h` instead.